forecast_solar/__init__.py

```diff
@@ -135,10 +136,13 @@ async def estimate(self) -> Estimate:
         Returns:
             A Estimate object, with a estimated production forecast.
         """
+        params = {"time": "iso8601", "damping": str(self.damping)}
+        if self.inverter is not None:
+            params["inverter"] = str(self.inverter)
```

Comment:
```suggestion
            params["inverter"] = int(self.inverter)
```

Reply:
Or at least this should be float?

Reply:
It should be a float, eg `4.5` if you have a 4.5kW / 4500W inverter

Reply:
Do you still adjust this to a float? Then it can be merged 😄

Reply:
I followed the pattern of kwp that was already in here for this library and my local HA code that utilises it - Watts in configuration and then `/ 1000` before sending to the library.

Reply:
It is already a `float`, so wrapping in `float()` would not do anything. 
If anything we could drop the `str()` wrapper altogether, I am not sure about the requests and just followed on from the wrapping of `str(self.damping)` which is also a `float` but cast to `str` for the params?
```suggestion
            params["inverter"] = self.inverter
```

Reply:
Then I think it will be fine.